app/src/main/assets/country_metadata/HR.yml

```
@@ -14,6 +14,7 @@ noParkingSignStyle: vienna
 noStoppingSignStyle: vienna
 officialLanguages: [hr]
 orchardProduces: [grape, olive, walnut, hazelnut, apple, plum, cherry, orange, peach, pear, almond, chilli_pepper, sweet_pepper, fig, tomato, blueberry]
+parcelLockerBrand: [Paket24, GLS, BoxNow]
```

```suggestion
parcelLockerBrand: [Paket24, GLS, BoxNow, DPD]
```

there are a [lot of DPD parcel lockers in Croatia](https://www.google.com/maps/d/viewer?mid=1-SldYNLgTpq5qwH-qq7TLO1NAfBFTFQ&ll=44.862215647602454%2C15.653692520409685&z=8) (not many are currently tagged in OSM, though, which is why this quest would help here) and it is listed at [nsi.guide](https://nsi.guide/index.html?t=brands&k=amenity&v=parcel_locker&cc=hr).

---

1. NSI suggests using "DPD Pickup Station" as brand (and "NSI" as operator).
2. I don't think it makes sense to manually change this file, as it will be overwritten anyway by the Gradle task before the next release.

---

This is autogenerated - either some POIs in Croatia miss this info and should be mapped or Gradle task need to be changed or service used here has some data problems.

This file definitely should not be edited manually.

---

> NSI suggests using "DPD Pickup Station" as brand (and "NSI" as operator).

You mean "DPD" as operator? Yes, I guess that was done to distinguish them. Either name would be fine with me...

> I don't think it makes sense to manually change this file, as it will be overwritten anyway by the Gradle task before the next release.

That is problematic then if it gets automatically overwritten, indeed. I guess the script ignores nsi.guide and relies on usage counts only? Could script be made to **_merge_** **both** "usage count" data **and** nsi.guide?  

The small annoyance is that it that might produce duplicates (e.g. theoretical _"DPD_ vs. _"DPD Pickup Station"_ in that example), but that seems better then not offering anything (especially when whole point of such quest is to add that data to the map!)
 
And "usage data" approach (sadly) produces much duplicates already (e.g. looking at ATM operators countrydata  we have in Croatia _"OTP Bank"_, _"OTP banka"_, _"OTP Banka"_, _"OTP"_, _"OTP banka d.d."_ etc.)

Perhaps NSI data could be put at the front of that merged list, as it more likely to be correct and more detailed (having `*:wikidata` etc). Should I open separate issue for that?

---

> And "usage data" approach (sadly) produces much duplicates already (e.g. looking at ATM operators countrydata we have in Croatia "OTP Bank", "OTP banka", "OTP Banka", "OTP", "OTP banka d.d." etc.)

This is just symptom of fragmented data in OSM - if one version is obviously preferable then bot edit can be applied. (if there is a clear consensus in community then running edit itself is easy, feel free to ping me in the community thread - but it is also easily doable with JOSM or level0)

---

@matkoniecz yeah it is quite easy to _actually do_ with overpass  & JOSM, it is just that the whole bureaucratic process of doing mechanical edits is so unappealing that I keep postponing it for [more two and a half years](https://github.com/streetcomplete/StreetComplete/pull/3352#issuecomment-939199676) now...

_And_ it does not solve this particular problem about parcel lockers operators, for which one would need to do an import (whose process is even more unappealing) or dedicate whole day or few targeted just to survey enough of them for script to pick them up (which is better used for other things)

Having the update script also **use (and ideally, prefer) NSI** in addition to taginfo usage would however be **much more preferable**, and would fix multiple of those issues (For example; I've already fixed ATM/banks in NSI: https://github.com/osmlab/name-suggestion-index/pull/9432, https://github.com/osmlab/name-suggestion-index/pull/9444, and improved parcel lockers in https://github.com/osmlab/name-suggestion-index/pull/9369) by itself...

---

For simple regional ones it as simple as asking on community forum (maybe also some other channel) and waiting two weeks whether anyone objects.

(I make also wiki pages even for obvious edits but that is easy to just copy past and amend some specific cases, though many people admittedly skip this part - and as long as you as ask before doing edit this is not so terrible, though I would not advise it)

---

Hm, this discussion is moot. At @matkoniecz noted:

> This is autogenerated

Nothing the PR author could do about without completely changing how this list is generated.